### ISSUE TITLE
Impossible to source the new belt spec in EU

### ISSUE THREAD

**Issue by testack18** [PrintersForAnts/Micron#133]
I have a brand new Formbot Micron+ kit. The new R1 specs mentions that we need 152mm belt but the Formbot kit still ships 188mm belt.

In the EU it is impossible to source genuine GATES belt with this spec. Even in 150mm length. It is also hard to find in stock for example in the US. Additionally if I find something in US just the shipping fee costs $90+ not included VAT. 

This means from my perspective that this belt spec is something which is hard to source. While Voron philosophy as far as I know is meant to be something which does not contain hardly sourcable or special components. I know Micron is not an official Voron, but.... You understand my point, right?

Anyways, any recommendation where to source genuine GATES belt with specs? Or any other good quality alternative within EU? I have read that the cheap aliexpress belts contains in practice steel instead of fiber glass (no matter what they states on their sales page) which makes them easily breakable compared to RF GATES belts. So any hints what can we do outside the US? Where can we source this component at a reasonable cost?
Or just what happens if I keep using the 188mm belt come with the Formbot kit?

Any hits/advice would be really appreciated!

**Comment by hartk1213** [comment]
@testack18 can you order from Ali Express? You can find them all over there...and for the closed loop belts I don't think getting official gates is as crucial as it is for the rest of the machine ..I've used non gates belts for those without issue

**Comment by testack18** [comment]
Of course, I can. My concern with them is what I mentioned I have read about them. Their reinforcements is not fiberglass whatever they states, which causes shorter lifetime. They weares out much faster then genuine GATES. I think I don't have to explain what unriabilities thise can cause after a (not so long) while. How long have you used those non gates belts you mentioned?
If you could share a link you have ordered, I would really appreciate!

Ohh, and I forgot to thank your huge effort you have put into this printer's design hartk!

**Comment by EllaFoxo** [comment]
Cutting down to size yourself should be viable if you buy per meter. [3DO](https://3do.dk/en/belts/279-original-gates-belts-gates-ll-2gt-price-per-meter.html) has it.
